ProteoformSuiteInternal/Lollipop.cs

```diff
@@ -513,10 +513,11 @@ public static void make_ee_relationships()
         //METHOD FILE
         public static string method_toString()
         {
-            return String.Join(System.Environment.NewLine, new string[] {
-                "LoadDeconvolutionResults|deconvolution_file_names\t" + String.Join("; ", Lollipop.identification_files().Select(s => s.filename).ToArray<string>()),
-                "LoadDeconvolutionResults|neucode_labeled\t" + neucode_labeled.ToString(),
-                "CorrectionFactors|correction_file_names\t" + String.Join("; ", Lollipop.calibration_files().Select(s => s.filename).ToArray<string>()),
+            string method = String.Join(System.Environment.NewLine, new string[] {
```

Comment:
I think in opening a full processed run, we should also display the file origins. Otherwise, the number of files and whether they were calibrated may not be clear.